lib/gui/modules/image-writer.js

```diff
@@ -102,12 +113,32 @@ imageWriter.service('ImageWriterService', function($q, $rootScope) {
         flashState.setProgressState(state);
       });
 
-    }).then(flashState.unsetFlashingFlag).catch((error) => {
+    }).then(flashState.unsetFlashingFlag).then(() => {
+      if (flashState.wasLastFlashCancelled()) {
+        AnalyticsService.logEvent('Elevation cancelled', analyticsData);
```

Comment:
What impact (if any) does this have on https://github.com/resin-io/etcher/pull/1344/files#diff-1e0579ca641fcc9114c426954e51435aR210 ?